libcxx/docs/ReleaseNotes/19.rst

```diff
@@ -47,6 +47,7 @@ Implemented Papers
 - P3142R0 - Printing Blank Lines with ``println`` (as DR against C++23)
 - P2302R4 - ``std::ranges::contains``
 - P1659R3 - ``std::ranges::starts_with`` and ``std::ranges::ends_with``
+- P0905R1 - Symmetry for spaceship (marked as implemented)
```

Comment:
Let's remove this release note since we didn't actually do anything new in this release.